source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/profile/profile.scss

```diff
@@ -83,11 +83,27 @@
     }
   }
 
-  .profile-image {
+  .thumbnail {
```

Comment:
do we need all of this?
have you seen? 
https://github.com/sillsdev/appbuilder-portal/blob/master/source/SIL.AppBuilder.Portal.Frontend/src/ui/styles/utility/image.scss#L1

it's what the settings page uses